src/js/control-bar/time-controls/current-time-display.js

```diff
@@ -39,19 +43,35 @@ class CurrentTimeDisplay extends Component {
     });
 
     this.contentEl_ = Dom.createEl('div', {
-      className: 'vjs-current-time-display',
-      // label the current time for screen reader users
-      innerHTML: '<span class="vjs-control-text">Current Time </span>' + '0:00'
+      className: 'vjs-current-time-display'
     }, {
       // tell screen readers not to automatically read the time as it changes
       'aria-live': 'off'
-    });
+    }, Dom.createEl('span', {
+      className: 'vjs-control-text',
+      textContent: this.localize('Current Time')
+    }));
 
+    this.updateTextNode_();
     el.appendChild(this.contentEl_);
     return el;
   }
 
   /**
+   * Updates the "current time" text node with new content using the
+   * contents of the `formattedTime_` property.
+   *
+   * @private
+   */
+  updateTextNode_() {
+    if (this.textNode_) {
+      this.contentEl_.removeChild(this.textNode_);
+    }
+    this.textNode_ = document.createTextNode(` ${this.formattedTime_}`);
```

Comment:
I think the former is preferable. I'll update. Can't believe I somehow didn't notice this...